cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -103,3 +103,17 @@ def test_projects_dashboard_icons(provider, appliance, soft_assert, container_pr
                 containers_cls.__name__, api_values[containers_cls], statusbox_value
             )
         )
+
+
+def test_project_has_provider(appliance, soft_assert):
```

Comment:
Required: Add doc string, with info what we are testing here

Reply:
@rlbabyuk doc string added